libcxx/include/__configuration/availability.h

```diff
@@ -359,6 +367,9 @@
 #define _LIBCPP_AVAILABILITY_HAS_FROM_CHARS_FLOATING_POINT _LIBCPP_INTRODUCED_IN_LLVM_20
 #define _LIBCPP_AVAILABILITY_FROM_CHARS_FLOATING_POINT _LIBCPP_INTRODUCED_IN_LLVM_20_ATTRIBUTE
 
+#define _LIBCPP_AVAILABILITY_HAS_HASH _LIBCPP_INTRODUCED_IN_LLVM_21
+// No attribute, since we've had hash in the headers before
```

Comment:
Please add a comment explaining what this availability macro is guarding.

Also, let's rename this to `_LIBCPP_AVAILABILITY_HAS_HASH_MEMORY` -- `HAS_HASH` is a bit too general IMO, it could be a bit confusing.